### PR TITLE
Downstream merge of scream (as of May 9, 2023) into E3SM

### PR DESCRIPTION
**Pull request by AaronDonahue**
Downstream merge of SCREAM development into E3SM - As of May 9, 2023

[BFB] (for E3SM cases)

### CODE REVIEW

**Comment by AaronDonahue** [comment]
@jgfouca and @ambrad I pinged you in this PR because the non-EAMxx changes are related to cime configs, particularly the bfbhash change and some crusher config changes.  The other machine file changes for Livermore machines quartz and ruby are from me and should be good for E3SM too.

**Review by ambrad — APPROVED** [review]

**Comment by rljacob** [comment]
Please change the title to something more descriptive and without the branch name.

File: cime_config/machines/config_machines.xml [hunk 1]
```diff
@@ -720,14 +720,14 @@
...
     <PROJECT>CLI133_crusher</PROJECT>
-    <CIME_OUTPUT_ROOT>/gpfs/alpine/cli133/proj-shared/$ENV{USER}/e3sm_scratch/crusher</CIME_OUTPUT_ROOT>
-    <DIN_LOC_ROOT>/gpfs/alpine/cli115/world-shared/e3sm/inputdata</DIN_LOC_ROOT>
-    <DIN_LOC_ROOT_CLMFORC>/gpfs/alpine/cli115/world-shared/e3sm/inputdata/atm/datm7</DIN_LOC_ROOT_CLMFORC>
+    <CIME_OUTPUT_ROOT>/lustre/orion/cli133/proj-shared/$ENV{USER}/e3sm_scratch/crusher</CIME_OUTPUT_ROOT>
+    <DIN_LOC_ROOT>/lustre/orion/cli115/world-shared/e3sm/inputdata</DIN_LOC_ROOT>
+    <DIN_LOC_ROOT_CLMFORC>/lustre/orion/cli115/world-shared/e3sm/inputdata/atm/datm7</DIN_LOC_ROOT_CLMFORC>
     <DOUT_S_ROOT>$CIME_OUTPUT_ROOT/archive/$CASE</DOUT_S_ROOT>
-       <CCSM_CPRNC>/gpfs/alpine/cli133/world-shared/e3sm/tools/cprnc/cprnc</CCSM_CPRNC>
+       <CCSM_CPRNC>/lustre/orion/cli133/world-shared/e3sm/tools/cprnc/cprnc</CCSM_CPRNC>
```
Comment (grnydawn) [inline cime_config/machines/config_machines.xml:730]: This path does not exist. Consider using "/lustre/orion/cli115/world-shared/e3sm/tools/cprnc/cprnc"

File: cime_config/machines/config_machines.xml [hunk 2]
```diff
@@ -826,11 +826,11 @@
...
     <PROJECT>CLI133_crusher</PROJECT>
-    <CIME_OUTPUT_ROOT>/gpfs/alpine/cli133/proj-shared/$ENV{USER}/e3sm_scratch/crusher</CIME_OUTPUT_ROOT>
-    <DIN_LOC_ROOT>/gpfs/alpine/cli115/world-shared/e3sm/inputdata</DIN_LOC_ROOT>
-    <DIN_LOC_ROOT_CLMFORC>/gpfs/alpine/cli115/world-shared/e3sm/inputdata/atm/datm7</DIN_LOC_ROOT_CLMFORC>
+    <CIME_OUTPUT_ROOT>/lustre/orion/cli133/proj-shared/$ENV{USER}/e3sm_scratch/crusher</CIME_OUTPUT_ROOT>
+    <DIN_LOC_ROOT>/lustre/orion/cli115/world-shared/e3sm/inputdata</DIN_LOC_ROOT>
+    <DIN_LOC_ROOT_CLMFORC>/lustre/orion/cli115/world-shared/e3sm/inputdata/atm/datm7</DIN_LOC_ROOT_CLMFORC>
     <DOUT_S_ROOT>$CIME_OUTPUT_ROOT/archive/$CASE</DOUT_S_ROOT>
-    <CCSM_CPRNC>/gpfs/alpine/cli133/world-shared/e3sm/tools/cprnc/cprnc</CCSM_CPRNC>
+    <CCSM_CPRNC>/lustre/orion/cli133/world-shared/e3sm/tools/cprnc/cprnc</CCSM_CPRNC>
```
Comment (grnydawn) [inline cime_config/machines/config_machines.xml:833]: This path does not exist. Consider using "/lustre/orion/cli115/world-shared/e3sm/tools/cprnc/cprnc"

**Review by grnydawn — APPROVED** [review]
E3SM inputdata is located at "/lustre/orion/cli115/world-shared/e3sm/inputdata" and only a group member of cli115 can use the auto-download feature of E3SM inputdata. So, please be aware that any non-cli115 user may have an issue of no-inputdata if the data file does not already exist in the E3SM inputdata path.

**Comment by jgfouca** [comment]
@grnydawn , good catch, thanks! I have pushed a fix.

**Comment by jgfouca** [comment]
Almost forgot about this. Pushed to next.

**Pull request merged**